sql/src/main/java/org/apache/druid/sql/calcite/expression/OperatorConversions.java

```diff
@@ -303,7 +303,7 @@ public static PostAggregator toPostAggregator(
             postAggregatorVisitor.getOutputNamePrefix() + postAggregatorVisitor.getAndIncrementCounter(),
             druidExpression.getExpression(),
             null,
-            plannerContext.getExprMacroTable()
+            plannerContext.parse(druidExpression.getExpression())
```

Comment:
nit: am conflicted, this reads funny like it isn't obvious at a glance it is parsing native expressions since like why would that be built-in to the plannerContext, otoh it would be hella tedious to always be `plannerContext.getExpressionParser().parse(...)` all the time..

Reply:
Yeah, the idea is to avoid doing `plannerContext.getExpressionParser().parse(...)` all the time.

What do you think about naming it `plannerContext.parseExpression(...)`?

Reply:
>What do you think about naming it plannerContext.parseExpression(...)?

that does seem nicer :+1: